small-talk/app/data/friendData/exampleFriendData.json

```diff
@@ -0,0 +1,38 @@
+[
+    {
```

Comment:
If you wish to expand upon this, you can import this json directly into a mongodb collection and make an api to begin making requests